test/tests/meshgenerators/poly2tri_mesh_generator/poly2tri_mesh_generator.i

```diff
@@ -0,0 +1,23 @@
+[Mesh]
+  [./outer_bdy]
+    type = PolyLineMeshGenerator
+    points = '-1.0 0.0 0.0
+              0.0 -1.0 0.0
+              1.0 0.0 0.0
+              0.0 2.0 0.0'
+    loop = true
+  []
+  [./triang]
+    type = Poly2TriMeshGenerator
+    boundary = 'outer_bdy'
+    interpolate_boundary = 4
+    refine_boundary = false
+    desired_area = 0.05
+  []
+
+  parallel_type = replicated  # libMesh bug workaround
```

Comment:
well will you really support distributed meshes?
seems like not to me even if poly2tri is/were distributed

we should have a check on this. use mesh.is_replicated or something like this

Reply:
Distributed meshes?  No.  `DistributedMesh`es?  Yes.  I threw a MeshSerializer in there, which ought to be fine for most "2D that won't get big until it's extruded to 3D" use cases.  I'm just not sure where or how it regressed; the poly2tri unit tests in libMesh ought to be running both replicated and distributed in CI.

Reply:
so this needs to stay?
can you please link to the issue there so I can read about it or if it s not fixed soon we can just keep track of it in the future

Reply:
The libMesh side of the fixes is going in via https://github.com/libMesh/libmesh/pull/3324

That plus the MOOSE side of the fixes is working for me, so I'm also removing all the `parallel_type = replicated` lines there.  I'll hold off on pushing and triggering CI here until we've got the next libMesh submodule update done too, so until then I'm mirroring to a `mesh_generators_distributed` branch.

Reply:
> That plus the MOOSE side of the fixes ~~is working for me~~

I should never type "is working for me" about new DistributedMesh code until I've finished a sweep over processor counts.  Let's say "is getting a tiny bit closer for me", at least.